src/python/pants/notes/2.16.x.md

```diff
@@ -14,6 +15,11 @@ the same syntax as target selectors.
 
 ### Backends
 
+#### Docker
+
+The [`{pants.hash}`](https://www.pantsbuild.org/docs/tagging-docker-images#string-interpolation-using-placeholder-values)
+generation code was changed such that the generated hash will change for the same input.
```

Comment:
This seems to suggest that the hash will always change given the same input. Maybe specify that it will change in 2.16 to clarify that it is still stable, but the algorithm is different in this version of pants.
```suggestion
generation code was changed such that the generated hash for the same input will be different in pants 2.16.
```

Reply:
Had the same thought reading that.